charts/astronomer/values.yaml

```diff
@@ -191,6 +191,39 @@ houston:
 
     # Default here is to run at midnight every night
     schedule: "0 0 * * *"
+  
+  # this option allow to overrides airflow releases config
```

Comment:
this option allows to override airflow releases config

Reply:
done.